ci/pipelines/ubuntu-jammy/stemcells.yml

```
@@ -0,0 +1,29 @@
+  ]
+  os: jammy
+  os_version: "22.04"
+  os_name: ubuntu-jammy
+  subnet_int: "3"
+
+blobstore_types:
+- dav
+- gcs
+- s3
+- azure-storage
+
```

YAMLlint (1.38.0)

[error] too many blank lines (1 > 0)

(empty-lines)